### ISSUE TITLE
Make `bean-test` artifact accessible again

### ISSUE THREAD

**Issue by StefanKock** [hzi-braunschweig/SORMAS-Project#9960]
### Problem Description

https://repository.novatec-gmbh.de/content/repositories/novatec is now down (responding with _503 Service Temporarily Unavailable_) since at least 24h. This blocks our tests in Github Actions, because artifact `info.novatec:bean-test` is only hosted in that repository.

### Proposed Solution

1. Contact the hosting company if the repo will come online again.
2. Consider pushing the existing artifact or a fork of it to Maven Central.

### Additional Information

- `info.novatec:bean-test` is the framework we use to tests most of our backend by emulating an environment where EJBs can be tested directly without extensive mocking.
- All build environments that cache maven artifacts still work as long as the local maven repository (or that specific artifact) is not wiped out.
- Related ticket in Source repo: https://github.com/NovatecConsulting/BeanTest/issues/29

**Comment by StefanKock** [comment]
I contacted Novatec via email to ask what happens with that Maven Repo.

**Comment by StefanKock** [comment]
> I contacted Novatec via email to ask what happens with that Maven Repo.

Reply from Novatec. The repository was shut down on purpose, and uploading `bean-test` somewhere else will likely not be done within the next days.

**Comment by StefanKock** [comment]
As discussed with @MartinWahnschaffe I tried to provide `bean-test` as system scope:
```xml
			<dependency>
				<groupId>info.novatec</groupId>
				<artifactId>bean-test</artifactId>
				<version>0.2.Final-Replaced</version>
				<scope>system</scope>
				<systemPath>${project.basedir}/../sormas-base/dependencies/bean-test-0.2.Final.jar</systemPath>
				<!-- Changed scope "test" to "system" as workaround for currently no publicly available maven artifact -->
			</dependency>
```

But this decouples all transitive dependencies and therefore does work with this minimalistic approach.

Another approach uses a self-published artifact version but needs a hardcoded access token if we don't want to force every developer and CI system to change the settings.xml.
```xml
<repository>
   <id>BeanTest</id>
   <name>BeanTest Repository</name>
   <url>https://username:accesstoken@maven.pkg.github.com/hzi-braunschweig/BeanTest</url>
   <!- - Contains access token to download from Github Packages (and only for that use case) - ->
</repository>
```

**Comment by StefanKock** [comment]
Now fixed on `development`, still needs to be fixed on `hotfix-1.74.1`.

**Comment by carina29** [comment]
This problem also appears for Sormas v 1.74 and v 1.73 which is preventing the Survnet 14.0 release.
![image](https://user-images.githubusercontent.com/47103965/181525919-3f0b2f3f-e675-4515-a33b-16cada8b633c.png

**Comment by MateStrysewske** [comment]
@MartinWahnschaffe Your last commit broke the build on Jenkins because the dependency can't be resolved there. I've commented the change and created a new issue in which we should properly fix this: #10101

**Comment by carina29** [comment]
@MateStrysewske  It was funny that we found with @JonasCir the problem at the same time when you committed the revert of the last commit. :) Without the change from serverlibs.pom everything was working fine.
